update/sync/syncTries.go

```diff
@@ -107,12 +108,12 @@ func (st *syncTries) SyncTriesFrom(meta *block.MetaBlock, waitTime time.Duration
 }
 
 func (st *syncTries) syncMeta(meta *block.MetaBlock) error {
-	err := st.syncTrieOfType(factory.UserAccount, core.MetachainShardId, meta.RootHash)
+	err := st.syncTrieOfType(factory.UserAccount, factoryTrieContainer.UserAccountTrie, core.MetachainShardId, meta.RootHash)
 	if err != nil {
 		return err
 	}
 
-	err = st.syncTrieOfType(factory.ValidatorAccount, core.MetachainShardId, meta.ValidatorStatsRootHash)
+	err = st.syncTrieOfType(factory.ValidatorAccount, factoryTrieContainer.PeerAccountTrie, core.MetachainShardId, meta.ValidatorStatsRootHash)
```

Comment:
Peer <> Valiator? Suggestion to have only one of them

Reply:
agreed. but would do it in development to not have too many merge conflicts.